src/extensions/additional/QuoteLink/QuoteLinkSpecs/index.ts

```diff
@@ -0,0 +1,75 @@
+import {transform as quoteLink} from '@diplodoc/quote-link-extension';
+import type {Node} from 'prosemirror-model';
+
+import type {ExtensionAuto} from '#core';
+import {nodeTypeFactory} from 'src/utils';
+
+import {moveLinkToQuoteAttributes} from './md/moveLinkToQuoteAttributes';
+
+export const quoteLinkNodeName = 'yfm_quote-link';
+export const quoteLinkType = nodeTypeFactory(quoteLinkNodeName);
+export const isQuoteLinkNode = (node: Node) => node.type.name === quoteLinkNodeName;
+
+export enum QuoteLinkAttr {
+    Href = 'href',
+    Content = 'content',
```

Comment:
element `<blockquote>` does not have `content` attribute, so attribute must be `data-*`